src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XElement.cs

```diff
@@ -131,7 +133,7 @@ public XElement(XStreamingElement other)
         }
 
         internal XElement()
-            : this("default")
+            : this("default"!)
```

Comment:
why is `!` needed here?

Reply:
A bug in Roslyn https://github.com/dotnet/roslyn/issues/39802.
This string goes to an implicit operator which does not recognize `NotNullIfNotNull` attribute.